scripts/extract_localizable.sh

```diff
@@ -9,25 +9,17 @@ LANGUAGES=( "Base" )
 
 for lang in ${LANGUAGES[@]}
 do
+    source "${DIR}/file_conversion.sh"
+
     echo "Extracting ${lang} strings"
 
     # Extract localizable strings from .swift files
     find ${NAVIGATION} -name "*.swift" -print0 | xargs -0 xcrun extractLocStrings -o "${NAVIGATION}/Resources/${lang}.lproj"
     STRINGS_FILE="${NAVIGATION}/Resources/${lang}.lproj/Localizable.strings"
+    convertIfNeeded "$STRINGS_FILE"
 
     # Extract localizable strings from .swift files
     find ${CORE} -name "*.swift" -print0 | xargs -0 xcrun extractLocStrings -o "${CORE}/Resources/${lang}.lproj"
     STRINGS_FILE="${CORE}/Resources/${lang}.lproj/Localizable.strings"
-
```

Comment:
I removed some strings, so I wanted to regenerate them. Unfortunately the script was broken, so I fixed that too.

It seems like maybe there were some other stale strings that hadn't been removed by earlier work.